mappings/net/minecraft/server/ServerReloadableResources.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/unmapped/C_migzkpst net/minecraft/server/ServerReloadableResources
+	FIELD f_bqclfysi LOGGER Lorg/slf4j/Logger;
+	FIELD f_cqvnljsk recipes Lnet/minecraft/unmapped/C_hjseusrb;
+	FIELD f_izcsyrwb functionLibrary Lnet/minecraft/unmapped/C_coevuhwv;
+	FIELD f_jpdcebur commands Lnet/minecraft/unmapped/C_rieemtgi;
+	FIELD f_nupmvkrd tagLoader Lnet/minecraft/unmapped/C_urxtamub;
+	FIELD f_pcwirfvq lootTables Lnet/minecraft/unmapped/C_chzdbgdx;
+	FIELD f_tcegvcns advancements Lnet/minecraft/unmapped/C_fktoyupt;
+	FIELD f_xgqmoaaw INITIAL_DATA_RELOAD_TASK Ljava/util/concurrent/CompletableFuture;
+	FIELD f_xpwimqwt lootConditions Lnet/minecraft/unmapped/C_gztbwabg;
+	FIELD f_zdhwgzem lootFunctions Lnet/minecraft/unmapped/C_wqgroktp;
+	METHOD m_arswxfre getCommands ()Lnet/minecraft/unmapped/C_rieemtgi;
+	METHOD m_bjfjrecc updateRegistryTags (Lnet/minecraft/unmapped/C_wqxmvzdq;Lnet/minecraft/unmapped/C_urxtamub$C_rbziadam;)V
+	METHOD m_hsxolvbj getAdvancements ()Lnet/minecraft/unmapped/C_fktoyupt;
+	METHOD m_qobkrmhz listeners ()Ljava/util/List;
+	METHOD m_qpiprbld getLootFunctions ()Lnet/minecraft/unmapped/C_wqgroktp;
+	METHOD m_qzzzeens getRecipes ()Lnet/minecraft/unmapped/C_hjseusrb;
+	METHOD m_ravmxbhg updateRegistryTags (Lnet/minecraft/unmapped/C_wqxmvzdq;)V
+	METHOD m_txfqcvol getLootConditions ()Lnet/minecraft/unmapped/C_gztbwabg;
+	METHOD m_xlzbyznn loadResources (Lnet/minecraft/unmapped/C_tmnrpasf;Lnet/minecraft/unmapped/C_wqxmvzdq$C_ggmzysrd;Lnet/minecraft/unmapped/C_rieemtgi$C_vypaxkxj;ILjava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
+		ARG 0 resources
+		ARG 2 environment
+		ARG 4 executor
+		ARG 5 executor2
```

Comment:
i would try to figure more about what these do. if there are just two then this is fine.